### PR TITLE
Added user icon image and set up 2 column display for Featured Articles

### PR DESCRIPTION
**Pull request by mmqw**
Added temp user icons and set up 2 column display of Featured Articles 

image

Connects #24

### CODE REVIEW

File: app/views/home/index.slim [hunk 1]
```diff
@@ -9,13 +9,27 @@ section.card.card-2.mx1.mb1
...
+
+  - if @posts.empty?
+    h2 There aren't any posts yet
+  - else
+    ul.list-reset.m0
+      - @posts.each do |post|
+        li.two-col
+          div.a.card-divider.block.text-decoration-none.py2 href="#{post_path(post)}"
+            div.table-cell.px1 style="height:80px;"
+              h5.h5.m0.card-listItem-header= post.title
```
Comment (gautamgupta) [inline app/views/home/index.slim]: Can you remove the space after `=`

File: app/views/home/index.slim [hunk 2]
```diff
@@ -9,13 +9,27 @@ section.card.card-2.mx1.mb1
...
+  h1.card-divider.h4.m0.p2.regular.left-align Featured Articles
+
+  - if @posts.empty?
+    h2 There aren't any posts yet
+  - else
+    ul.list-reset.m0
+      - @posts.each do |post|
+        li.two-col
+          div.a.card-divider.block.text-decoration-none.py2 href="#{post_path(post)}"
+            div.table-cell.px1 style="height:80px;"
```
Comment (gautamgupta) [inline app/views/home/index.slim]: Maybe move the inline style to a css file? :)

File: app/frontend/common/components/card/card.less [hunk 3]
```diff
@@ -16,6 +17,16 @@
     }
   }
 
+  .columns{
```
Comment (gautamgupta) [inline app/frontend/common/components/card/card.less]: Space before {

File: app/frontend/common/components/card/card.less [hunk 4]
```diff
@@ -1,4 +1,5 @@
 @import "../../colors";
+//@import "compass/css3";
```
Comment (gautamgupta) [inline app/frontend/common/components/card/card.less]: We can take this line out

File: app/frontend/common/components/card/card.less [hunk 5]
```diff
@@ -16,6 +17,16 @@
     }
   }
 
+  .columns{
+    float:left;
```
Comment (gautamgupta) [inline app/frontend/common/components/card/card.less]: Space after `:` here and below

File: app/frontend/common/components/card/card.less [hunk 6]
```diff
@@ -27,6 +38,11 @@
   }
 }
 
+li.columns {
+  display: inline-block;
+  width: 50%
```
Comment (gautamgupta) [inline app/frontend/common/components/card/card.less]: Missing `;`

File: app/views/home/index.slim [hunk 7]
```diff
@@ -9,13 +9,27 @@ section.card.card-2.mx1.mb1
...
               p.m0.card-listItem-text=TruncateContent(post.content)
 
   div.right-align.py2.pr2
     a.btn.btn-secondary.mr2 Visit Forum
     button.btn Join The Community
 
 section#HomeSearchComponent.homeSearch.card.card-2.mx1.mtn1.mb2.__react-root
+
+section.card.card-2.mx1.mb1
+  h1.card-divider.h4.m0.p2.regular.left-align Featured Articles
```
Comment (gautamgupta) [inline app/views/home/index.slim]: Let's name it Featured Posts

**Review by gautamgupta — COMMENTED** [review]
Minor changes! :)

File: app/frontend/common/components/card/card.less (hunk 5 above)
Comment (mmqw) [reply app/frontend/common/components/card/card.less]: Don't actually need this class anymore, deleted.

**Comment by gautamgupta** [comment]
@mmqw Two comments left. Lmk when you're done :)

**Review by gautamgupta — APPROVED** [review]

File: app/views/home/index.slim [hunk 8]
```diff
@@ -9,13 +9,27 @@ section.card.card-2.mx1.mb1
         li
           a.card-divider.block.text-decoration-none.py2 href="#{post_path(post)}"
             div.table-cell.align-middle.px1
-              img style="height: 50px; width: 50px" src="https://startbootstrap.com/img/brand-logos/grav-logo.svg"
+              img.circle.user-icon src="https://i.pinimg.com/564x/7c/c7/a6/7cc7a630624d20f7797cb4c8e93c09c1.jpg"
```
Comment (gautamgupta) [inline app/views/home/index.slim:12]: We should store this locally in our assets folder.

**Pull request merged**